### DISCUSSION TITLE
Multiranger data

### DISCUSSION THREAD

**Discussion by liu-94** [USC-ACTLab/crazyswarm#601] [Q&A] [answered]
I am new to crazyswarm and using multiranger for my project,
may i know how to pull the multiranger data and use it in my script.

enabled logging for multiranger parameters in swarm_hover.launch, and have no idea how to proceed from here

**Comment by whoenig** [comment, marked as answer]
Select the logging variables from the firmware and put them in at https://github.com/USC-ACTLab/crazyswarm/blob/f177308a062397c1f7becfa6e9f2d1a219bf65e9/ros_ws/src/crazyswarm/launch/hover_swarm.launch#L13. Then you can get the output at the topic `/cf<ID>/log1`.
